<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSettingsWidget</class>
 <widget class="QWidget" name="SystemSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="pbApply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="deviceLayout"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbAddDev">
     <property name="text">
      <string> Device</string>
     </property>
     <property name="icon">
      <iconset resource="../../app/rc.qrc">
       <normaloff>:/images/ActionsListAdd.png</normaloff>:/images/ActionsListAdd.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblSystemTitle">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../app/rc.qrc"/>
 </resources>
 <connections/>
</ui>
